BREAKING-CHANGES.md

```diff
@@ -11,6 +11,7 @@ github with a [breaking change] label.
 This is a quick summary of the sections below:
 
 - [v0.24.0](#v0240)
+  - MSRV is now 1.70.0
```

Comment:
Was this change made after `v0.24.0` was released?

Reply:
@kdheepak https://github.com/ratatui-org/ratatui/pull/487